pydfs_lineup_optimizer/lineup_exporter.py

```diff
@@ -36,3 +37,37 @@ def export(self, filename, render_func=None):
                 row.append(str(lineup.salary_costs))
                 row.append(str(lineup.fantasy_points_projection))
                 lineup_writer.writerow(row)
+
+
+class JSONLineupExporter(LineupExporter):
+    def export(self):
+        totalLineups = []
+
+        for lineup in enumerate(self.lineups):
+            lineupList = []
+
+            # Generate players JSON object
+            for player in lineup.lineup:
+                playerJSON = {
+                    "id": player.id,
+                    "firstName": player.first_name,
+                    "lastName": player.last_name,
+                    "positions": player.positions,
+                    "team": player.team,
+                    "salary": player.salary,
+                    "fppg": player.fppg,
+                    "gameInfo": player.game_info
```

Comment:
game_info is a object of type GameInfo, so it should be converted to dict as well